src/Sprites/BmpHeader.cpp

```diff
@@ -1,10 +1,10 @@
 #include "BmpHeader.h"
 
-const std::array<char, 2> BmpHeader::defaultType{ 'B', 'M' };
+const std::array<char, 2> BmpHeader::defaultFileSignature{ 'B', 'M' };
```

Comment:
If we remove default, the static 'default' value would have the same name as the actual instance variable fileSignature. I thought about BmpFileSignature, but this sort of gets weird when you have the options BmpHeader::FileSignature and BmpHeader::BmpFileSignature as they are too similar.

Reply:
What about using Pascal Case (ake UpperCamelCase) for constants? Then it would be `FileSignature`, which won't conflict with `fileSignature` due to case sensitivity.

Reply:
A little, yes. I think UpperCamelCase is what I've already been using for constants.

Reply:
I just checked through the codebase, and it does appear we were using UpperCamelCase fairly exclusively before this branch. We should update this code to match.